document/core/valid/instructions.rst

```diff
@@ -440,8 +440,6 @@ Atomic Memory Instructions
 
 * Let :math:`\limits~\share` be the :ref:`memory type <syntax-memtype>` :math:`C.\CMEMS[0]`.
 
-* The sharedness :math:`\share` must be |MSHARED|.
```

Comment:
You'll also need to adjust the respective premise in the formal rule below to keep it in sync, e.g., by changing `C.\CMEMS[0] = \limits~\MSHARED` to `C.\CMEMS[0] = \memtype` (here and elsewhere).